metagpt/provider/bedrock/amazon_bedrock_api.py

```diff
@@ -0,0 +1,136 @@
+from typing import Literal
```

Comment:
put the bedrock_api.py in metagpt/provider/bedrock_api.py? and keep others inside bedrock folder.